apps/audio-loader/src/services/audio-loader-service-impl.ts

```
@@ -0,0 +1,63 @@
+import { PassThrough } from 'node:stream';
+import type { MessageInitShape } from '@bufbuild/protobuf';
+import { pipeline } from 'stream';
+import { spawn } from 'child_process';
+import ffmpegTransform from '../utils/ffmpeg-transform';
+
+export class AudioLoaderServiceImpl
+  implements ServiceImpl<typeof AudioLoaderService>
+{
+  async *downloadAudio(
+    { url, format }: DownloadAudioRequest,
+    {}: HandlerContext,
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)